python/cudf/cudf/utils/ioutils.py

```diff
@@ -962,9 +962,16 @@ def get_filepath_or_buffer(
         # fsspec does not expanduser so handle here
         path_or_data = os.path.expanduser(path_or_data)
 
-        fs, _, paths = fsspec.get_fs_token_paths(
-            path_or_data, mode=mode, storage_options=storage_options
-        )
+        try:
+            fs, _, paths = fsspec.get_fs_token_paths(
+                path_or_data, mode=mode, storage_options=storage_options
+            )
+        except ValueError as e:
+            if str(e).startswith("Protocol not known"):
+                return path_or_data, compression
+            else:
+                raise e
```

Comment:
Note I wasn't sure if this should just be `raise e` or `raise e from None`. I couldn't remember if raising a caught error has different behavior than just say `raise ValueError`.

Reply:
It works if you just raise `e`

```
def sam():
    try:
        1/0

    except ZeroDivisionError as e:
        if str(e).startswith("Something"):
            return "ValueError"
        else:
            raise e

print (sam())
```

```
Traceback (most recent call last):
  File "sam.py", line 11, in <module>
    print (sam())
  File "sam.py", line 9, in sam
    raise e
  File "sam.py", line 3, in sam
    1/0
ZeroDivisionError: division by zero

```